packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-
+import getStatusBadgeConfig from '@deriv/account/src/Configs/get-status-badge-config';
```

Comment:
Why don't you put the `getStatusBadgeConfid` somewhere that is shared throughout the whole project ? We'll focus on removing the dependancies of CFD package in our refactoring, so it would be better to avoid adding more at this stage.

Reply:
@hirad-deriv Thanks for the suggestion.  It requires quite some refactoring including files in wallet package, Can we do this refactoring later in a different PR?

Reply:
Sure Amina. Please leave a comment on top of this import so that we do not miss it later.